about.html

```diff
@@ -64,7 +64,7 @@ <h1 class="main-title">Blockchain World Conference 2022</h1>
             <small>
                 Please contact us per email for further questions 
                 about the Blockchain World Conference 2022<br>
-                <strong>bcsummit2022@bcaustralia.org</strong>
+                <strong>bcconference2022@bcaustralia.org</strong>
             </small>
         </div>
         </section>
```

Comment:
The section right below is missing the divider below the header.

<img width="912" alt="Screen Shot 2022-07-27 at 9 20 25" src="https://user-images.githubusercontent.com/82426682/181271116-7c1f2377-6747-48e0-abd3-d6229e98e6eb.png">